### PR TITLE
Fixed Alignment of Bundle Hover Info

### PR DESCRIPTION
**Pull request by Moonwolf287**
- moved background down to suround the text
- moved Icon up to be next to text & not overlap with gold price in tooltip

### CODE REVIEW

**Comment by Moonwolf287** [comment]
Original:
grafik
Updated:
grafik

**Comment by Moonwolf287** [comment]
Fix for #43 
grafik
grafik

File: SDVModTest/UIElements/ShowItemHoverInformation.cs [hunk 1]
```diff
@@ -208,11 +208,17 @@ private void DrawAdvancedTooltip()
...
+                if (!String.IsNullOrEmpty(requiredBundleName))
+                {
+	                bundleTextWidth = (int)Game1.dialogueFont.MeasureString(requiredBundleName).Length();
+	                bundleTextWidth -= 30; //Text offset from left
+                }
+				int stackTextWidth = (int)(Game1.smallFont.MeasureString(stackPrice.ToString()).Length());
                 int itemTextWidth = (int)(Game1.smallFont.MeasureString(itemPrice.ToString()).Length());
-                largestTextWidth = (stackTextWidth > itemTextWidth) ? stackTextWidth : itemTextWidth;
-                int windowWidth = Math.Max(largestTextWidth + 90, String.IsNullOrEmpty(requiredBundleName) ? 100 : 300);
+	            int largestTextWidth = Math.Max(bundleTextWidth,Math.Max(stackTextWidth, itemTextWidth));
```
Comment (cdaragorn) [inline SDVModTest/UIElements/ShowItemHoverInformation.cs]: Please remove tabs and keep spacing consistent with what I'm using (4 spaces per tab)

**Comment by Annosz** [comment]
Hello @Moonwolf287! I'm trying to create a new version of UI Info Suite and support maintainance for it in a new fork. I want to create a stable release for Stardew Valley 1.4, and I want to include unmerged pull requests from this repository.

I'd like to merge you PRs too, but I want to give you the chance to create them yourself to get the contribution :) If you want, you can recreate your pull requests [in this fork](https://github.com/Annosz/UIInfoSuite), or if you do not want to spend time with it, I will do it and mention you in a contributors section on the mod page.

**Comment by Zamiell** [comment]
This PR has been integrated in Annosz's fork: https://github.com/Annosz/UIInfoSuite2

**Pull request merged**